apps/E2E/src/Menu/specs/Menu.spec.win.ts

```diff
@@ -53,27 +74,27 @@ describe('Menu Accessibility Testing', () => {
 describe('Menu Functional Testing', () => {
   /* Scrolls and waits for the Menu to be visible on the Test Page */
   beforeEach(async () => {
-    await MenuPageObject.scrollToTestElement();
+    await MenuPageObject.scrollToTestElement(await MenuPageObject._menuTrigger);
 
     await MenuPageObject.resetTest();
   });
 
   it('Click MenuTrigger -> Validate Menu is opened by checking if MenuItems are visible', async () => {
-    await MenuPageObject.clickItem(MenuComponentSelector.MenuTrigger);
+    await MenuPageObject.click(MenuPageObject._menuTrigger);
     await expect(await MenuPageObject.waitForMenuToOpen()).toBeTruthy();
```

Comment:
Add error message to waiter

Reply:
For this specific test case, if the waiter fails, it returns the error `The Menu did not open`. Combined with the name of the case itself, would this be clear enough? Adding an extra error feels a bit redundant.